MonoZelda/Link/Projectiles/Projectile.cs

```diff
@@ -0,0 +1,104 @@
+﻿using PixelPushers.MonoZelda.Sprites;
+using Microsoft.Xna.Framework;
+using PixelPushers.MonoZelda.Commands;
+using System;
+
+namespace PixelPushers.MonoZelda.Link.Projectiles;
+
+public class Projectile
+{
+
+    private ProjectileType currentProjectile;
+    private SpriteDict projectileDict;
+    private Player player;
+    protected Vector2 projectilePosition;
+    protected Direction playerDirection;
+
+    public ProjectileType CurrentProjectile
+    {
+        get
+        {
+            return currentProjectile;
+        }
+        set
+        {
+            currentProjectile = value;  
+        }
+    }
+
+    public Projectile(SpriteDict projectileDict,Player player)
```

Comment:
If we wanted to avoid coupling the player to all projectiles, instead of player, we could pass a position.